.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature
+about: Suggest an idea for this project
+title: "[FEATURE]"
```

Comment:
What's the difference between name and title?

Reply:
Name = the name of the template so you'll see that in the dropdown list

Title = Default pre-filled title before one is added :)